pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetDsmKitRequestsRoute.java

```diff
@@ -22,15 +25,23 @@
     public Object handle(Request request, Response response) {
         return TransactionWrapper.withTxn(handle -> {
             String studyGuid = request.params(STUDY_GUID);
-            if (studyGuid == null) {
```

Comment:
This is not a useful check -- if study guid is missing SparkJava won't match the URL and this handler won't even be executed. I think it's safe to assume it's there.